### PR TITLE
Update anchor to 0.20

### PR DESCRIPTION
**Pull request by archseer**

### CODE REVIEW

File: gauntlet/packages/gauntlet-solana/package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
   },
   "dependencies": {
     "@chainlink/gauntlet-core": "0.0.7",
-    "@project-serum/anchor": "^0.18.0",
+    "@project-serum/anchor": "^0.20.0",
```
[the remark this answers is not in the document]
Comment (krebernisak) [reply gauntlet/packages/gauntlet-solana/package.json:29]: I've opened a GH Issue:

- https://github.com/smartcontractkit/chainlink-solana/issues/93

Please just append the Gauntlet changes to this PR...

[the remark this answers is not in the document]
Comment (aalu1418) [reply gauntlet/packages/gauntlet-solana/package.json:29]: Seems like this is what is happening, but i'm not sure how to solve it as the proposed solution doesn't quite work 🤔 
https://stackoverflow.com/a/63639280

File: gauntlet/packages/gauntlet-solana/src/commands/internal/solana.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { TransactionResponse } from '../types'
 import { Idl, Program, Provider, Wallet } from '@project-serum/anchor'
 
 export default abstract class SolanaCommand extends WriteCommand<TransactionResponse> {
-  wallet: Wallet
+  wallet: typeof Wallet
```
Comment (aalu1418) [inline gauntlet/packages/gauntlet-solana/src/commands/internal/solana.ts:8]: Fix 1

File: gauntlet/packages/gauntlet-solana/src/commands/middlewares.ts [hunk 3]
```diff
@@ -1,8 +1,9 @@
 import { Middleware, Next } from '@chainlink/gauntlet-core'
 import { assertions } from '@chainlink/gauntlet-core/dist/utils'
-import { Provider, Wallet } from '@project-serum/anchor'
+import { Provider } from '@project-serum/anchor'
 import { Connection, Keypair } from '@solana/web3.js'
 import SolanaCommand from './internal/solana'
+const { Wallet } = require('@project-serum/anchor') // module exported dynamically from anchor
```
Comment (aalu1418) [inline gauntlet/packages/gauntlet-solana/src/commands/middlewares.ts:6]: Fix 2

**Review by aalu1418 — COMMENTED** [review]
These fixes seem to get it to work. Tested it with a `deploy:token` command. Tested with local env ✅

**Review by krebernisak — APPROVED** [review]
LGTM!

**Pull request merged**